actors/crypto/cbor_gen.go

```diff
@@ -105,3 +105,32 @@ func (t *Signature) UnmarshalCBOR(r io.Reader) error {
 	}
 	return nil
 }
+
+func (t *AddressEpochEntropy) MarshalCBOR(w io.Writer) error {
+	if t == nil {
+		_, err := w.Write(cbg.CborNull)
+		return err
+	}
+	if _, err := w.Write([]byte{128}); err != nil {
```

Comment:
Im a bit new to cbg but it looks like this just writes 0x80 and calls it a day without writing fields.